src/views/DandisetLandingView/Meditor.vue

```diff
@@ -249,8 +245,11 @@ export default defineComponent({
         );
 
         if (status === 200) {
-          setDandiset(data);
-          closeEditor();
+          // wait 0.5 seconds to give the celery worker some time to finish validation
+          setTimeout(async () => {
+            await store.dispatch('dandiset/fetchPublishDandiset', { identifier: data.dandiset.identifier, version: data.version });
+            closeEditor();
+          }, 500);
```

Comment:
I think this is the place where we would have to do what you described in your PR description. Could you file an issue to change the application behavior to have an intermediate validation state (something like `computing`) that the frontend can use to display a throbber or some sort in place of the error list?

Reply:
I just realized there's already [a flag like that](https://github.com/dandi/dandi-api/blob/master/dandiapi/api/models/version.py#L70) implemented, so this will just be a matter of adding that field to a django serializer and making use of it in the frontend. I filed an issue for the frontend changes in #830.